gulpfile.js

```diff
@@ -79,14 +85,30 @@ gulp.task("watch-styles", ["default"], () => {
 // Default to clean and build
 
 gulp.task("default", () =>
-    runSequence("clean", "build"));
+    runSequence("clean", "test", "build"));
```

Comment:
I say fuck eslint, unless we remove formatting bullshit. "Lint" means "look for logic errors" not "bitch at the programmer until the code fits on a printer from 1978"